library/alloc/src/vec/mod.rs

```diff
@@ -1001,7 +1001,7 @@ impl<T, A: Allocator> Vec<T, A> {
     /// If `len` is greater than the vector's current length, this has no
     /// effect.
     ///
-    /// The [`drain`] method can emulate `truncate`, but causes the excess
+    /// The [`split_off`] is similar to `truncate`, but causes the excess
```

Comment:
```suggestion
    /// The [`split_off`] method is similar to `truncate`, but causes the excess
```

maybe further reword the sentence, calling the tail "excess elements" feel very strange.

Reply:
Sounds good. I'll change the wording to "tail elements".